app/views/jst/workspace/search/embed_code.jst

```diff
@@ -11,6 +11,9 @@
     <%= options %>
   });
 &lt;/script&gt;
+&lt;noscript&gt;
+  &lt;a href="<%= window.location %>"&gt;View/search document collection&lt;/a&gt;
```

Comment:
So... this would link back to the *workspace* not to the public search, or to the page the search was embedded on.  So i feel like this is a little weird.

I'm not certain whether it makes more sense to link to the public archive or not.